auth/api/http/policies/transport.go

```diff
@@ -33,6 +33,13 @@ func MakeHandler(svc auth.Service, mux *bone.Mux, tracer opentracing.Tracer) *bo
 		opts...,
 	))
 
+	mux.Put("/policies", kithttp.NewServer(
+		kitot.TraceServer(tracer, "delete_policies")(deletePoliciesEndpoint(svc)),
+		decodeCreatePolicyRequest,
```

Comment:
Why CreatePolicy and not DeletePolicy?

Reply:
Both creating and deleting operations use same structure which consists of `token` and same request body. Therefore, I used the same function. But I can make separate function to prevent anyd inconvenience or inconsistency

Reply:
it is confusing in the code... We should add a comment here (this what you clarified to me)